hack/base-images/containerbuild.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
In a comment can you add where you borrowed this script from?

Reply:
Added the link to the script.